### ISSUE TITLE
Persist most recent tinker color on all edit views

### ISSUE THREAD

**Issue by toolness** [mozilla/webmaker-core#188]
This issue has been migrated from mozilla/webmaker-android#2204.

It was originally written by [xmatthewx](https://api.github.com/users/xmatthewx) on Fri Jun 12 2015 and had the following description:

> Scenario: user adds a custom color for a header on a page. They want to use this color on other pages as well. They need to recreate the color using tinker mode. (I guess it gives them a reason to memorize hex or rgb values). Ideally, most recent tinker color persists in all edit views. 

On Sat Jun 13 2015, [LauraReynal](https://api.github.com/users/LauraReynal) commented:

> I would tend to see if they memorize the numbers. 
> 
> I feel the same thing when I play with tinker mode, but I like the challenge, as a user.

**Comment by xmatthewx** [comment]
related to #740

**Comment by alanmoo** [comment]
With new elements soon defaulting their style to the last edited element of the same type (within a project) from #740, I'm not sure this is as much of an issue anymore. To be fair though, I'm still a bit unclear about what the expected behavior of this would be- currently the edit mode color picker shows the default colors and the currently selected color if it was manually chosen via tinker. Is this issue asking to always show an extra color, which changes based on any tinker mode selection in the app/project? Or for elements of the same type? What if the element being edited is already a custom color, but we were recently in another tinker view with a different color? Would we show 2 extra color circles at that point (which might get a bit crowded)?

**Comment by xmatthewx** [comment]
You're right that default styles will make this need less urgent. We can de-prioritize this. 

The original idea was to remember a handful of custom colors, a second row in the color picker everywhere that it appears. The revised idea was to save a single, most recent tinker color. This would allow a user to apply it to both text and image borders, for example. 

Persisting styles removes the biggest pain point. We can bump this out of the milestone.

**Comment by xmatthewx** [comment]
Before any implementation, a new design spec based on the apps evolution is required.
